src/Loader.php

```diff
@@ -812,10 +808,8 @@ public static function embed_page_header() {
 		<div id="woocommerce-embedded-root" class="is-embed-loading">
 			<div class="woocommerce-layout">
 				<div class="woocommerce-layout__header is-embed-loading">
-					<h1 class="woocommerce-layout__header-breadcrumbs">
-						<?php foreach ( $sections as $section ) : ?>
-							<?php self::output_breadcrumbs( $section ); ?>
-						<?php endforeach; ?>
+					<h1 class="woocommerce-layout__header-heading">
+						<?php self::output_breadcrumbs( end( $sections ) ); ?>
```

Comment:
Nitpick: we can rename this to something like `output_header_title()` to remove references to breadcrumbs.

Reply:
Good call; I did this elsewhere but forgot this one!